models/tokens/optimism/reaper_all_vaults.sql

```diff
@@ -0,0 +1,66 @@
+{{ config( alias='erc20')}}
+SELECT LOWER(contract_address) AS contract_address, symbol, decimals
+  FROM (VALUES 
+
+,('0x39FdE572a18448F8139b7788099F0a0740f51205','OATH', 18)
```

Comment:
Remove the extra comma here then it should be able to pass the dbt run

Reply:
@0xToli any reason this view needs to exist if you add the same tokens to the existing erc20 optimism view below?

Reply:
please note merge of #1783 -- i believe that makes this add here irrelevant